bundles/model/org.eclipse.smarthome.model.thing.tests/src/test/java/org/eclipse/smarthome/model/thing/test/hue/TestHueChannelTypeProvider.java

```diff
@@ -58,10 +59,12 @@ public TestHueChannelTypeProvider() {
                     new URI("Xhue", "XLCT001:Xcolor_temperature", null));
             channelTypes = Arrays.asList(ctColor, ctColorTemperature, ctColorX, ctColorTemperatureX);
 
-            ChannelGroupType groupX = new ChannelGroupType(GROUP_CHANNEL_GROUP_TYPE_UID, false, "Channel Group",
-                    "Channel Group", null,
-                    Arrays.asList(new ChannelDefinition("foo", TestHueChannelTypeProvider.COLOR_CHANNEL_TYPE_UID),
-                            new ChannelDefinition("bar", TestHueChannelTypeProvider.COLOR_CHANNEL_TYPE_UID)));
+            ChannelGroupType groupX = ChannelGroupTypeBuilder.instance(GROUP_CHANNEL_GROUP_TYPE_UID, "Channel Group")
+                    .isAdvanced(false).withDescription("Channel Group")
```

Comment:
`isAdvanced(false)` is default for `ChannelGroupTypeBuilder`. We can skip this one.